modules/apps/calendar/calendar-web-test/src/testFunctional/tests/CalendarPage.testcase

```diff
@@ -157,9 +157,9 @@ definition {
 
 		CalendarEvent.disableEventBySiteCalendarName(calendarName = "Liferay DXP");
 
-		CalendarScheduler.viewSpecificEventNotPresent(eventTitle = "Calendar Event Title 1");
+		CalendarScheduler.viewSpecificEventNotPresent(eventTitle = "Calendar Event Title 2");
 
-		CalendarScheduler.viewSpecificEventPresent(eventTitle = "Calendar Event Title 2");
+		CalendarScheduler.viewSpecificEventPresent(eventTitle = "Calendar Event Title 1");
```

Comment:
The user **Test Test** creates _Event 1_. The user **Liferay DXP** creates _Event 2_.
If I, as a user, disable the calendar of the **Liferay DXP** user, I will certainly not be able to view _event 2_.

Reply:
This one looks good to me